### PR TITLE
Feat/Desktop comment drawer

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/components/CommentBeta/Feed/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    margin-left: var(--spacing-base);
+
+    & > * + * {
+      margin-left: var(--spacing-tight);
+    }
+  }
+}
+
+.contentContainer {
+  width: 22.5rem;
```
Comment (robertu7) [inline src/components/CommentBeta/Feed/styles.module.css:52]: hardcoded?

File: src/components/CommentBeta/RoleLabel/index.tsx
Comment (robertu7) [inline src/components/CommentBeta/RoleLabel/index.tsx:1]: make a new component `<Label>` in https://github.com/thematters/matters-web/pull/4175/files#diff-ae6c67fe4fcac7f4dcc3460bb2b75856fb91006f00dd172528601935c3a31880, can reuse it later

File: src/components/Forms/CommentFormBeta/index.tsx
Comment (robertu7) [inline src/components/Forms/CommentFormBeta/index.tsx:1]: same as above, should be replace the `<CommentForm>` 

ref: [discussion](https://matterslab.slack.com/archives/G8877EQMS/p1707314475183879?thread_ts=1707305323.343669&cid=G8877EQMS)

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: lang/zh-Hans.json [hunk 2]
```diff
@@ -386,6 +386,10 @@
     "defaultMessage": "确认删除后，{commentType}会立即消失。",
     "description": "src/components/Comment/DropdownActions/DeleteComment/Dialog.tsx"
   },
+  "78qsg6": {
+    "defaultMessage": "Author",
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply lang/zh-Hans.json:390]: Has fixed in [next pr](https://github.com/thematters/matters-web/pull/4193)

File: lang/zh-Hans.json [hunk 3]
```diff
@@ -1748,6 +1790,10 @@
     "defaultMessage": "喜欢回应",
     "description": "src/components/Comment/DropdownActions/PinButton.tsx"
   },
+  "jc3mW/": {
+    "defaultMessage": "Supporter",
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply lang/zh-Hans.json:1794]: Has fixed in https://github.com/thematters/matters-web/pull/4193

File: lang/zh-Hant.json [hunk 4]
```diff
@@ -386,6 +386,10 @@
     "defaultMessage": "確認刪除後，{commentType}會立即消失。",
     "description": "src/components/Comment/DropdownActions/DeleteComment/Dialog.tsx"
   },
+  "78qsg6": {
+    "defaultMessage": "Author",
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply lang/zh-Hant.json:390]: Has fixed in https://github.com/thematters/matters-web/pull/4193

File: lang/zh-Hant.json [hunk 5]
```diff
@@ -1748,6 +1790,10 @@
     "defaultMessage": "喜歡回應",
     "description": "src/components/Comment/DropdownActions/PinButton.tsx"
   },
+  "jc3mW/": {
+    "defaultMessage": "Supporter",
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply lang/zh-Hant.json:1794]: Has fixed in https://github.com/thematters/matters-web/pull/4193

File: src/components/CommentBeta/Feed/styles.module.css (hunk 1 above)
Comment (pitb2022) [reply src/components/CommentBeta/Feed/styles.module.css:52]: yes

File: lang/zh-Hant.json [hunk 6]
```diff
@@ -1698,6 +1736,10 @@
     "defaultMessage": "人追蹤",
     "description": "src/views/TagDetail/Followers/index.tsx"
   },
+  "hgjjhO": {
+    "defaultMessage": "Comment",
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply lang/zh-Hant.json]: fixed

File: src/components/CommentBeta/RoleLabel/index.tsx
Comment (pitb2022) [reply src/components/CommentBeta/RoleLabel/index.tsx:1]: Has added todo note, will reuse it later

File: lang/en.json
Comment (robertu7) [inline lang/en.json:1]: image

File: src/components/CommentBeta/Feed/index.tsx [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+            <section className={styles.info}>
+              <section className={styles.top}>
+                <LinkWrapper {...userProfilePath}>
+                  <section className={styles.displayName}>
+                    {author.displayName}
+                  </section>
+                </LinkWrapper>
+                <RoleLabel comment={comment} />
+              </section>
+              <DateTime date={comment.createdAt} color="grey" />
```
Comment (robertu7) [inline src/components/CommentBeta/Feed/index.tsx:90]: there is link with hash (`#`) in legacy component, pls confirm with product if keep it or not

**Pull request merged**